multibody/parsing/detail_sdf_parser.cc

```diff
@@ -446,9 +446,12 @@ std::string LoadSdf(
   data_source.DemandExactlyOne();
 
   std::string root_dir;
+  sdf::ParserConfig parser_config = sdf::ParserConfig::GlobalConfig();
+  parser_config.SetWarningsPolicy(sdf::EnforcementPolicy::ERR);
```

Comment:
Without this, a non-empty `//inertial/pose/@relative_to` would generate a warning instead of an error. Regardless of this setting, libsdformat11 now removes the user specified `//inertial/pose/@relative_to` because it's recognized as invalid input. Thus, this code: https://github.com/jwnimmer-tri/drake/blob/aea9b8f4cf3eac863f7577f7c89776ede8d5ada7/multibody/parsing/detail_sdf_parser.cc#L65-L76 will not be able to detect if the user specified `//inertial/pose/@relative_to`